pkg/cmd/cli/cmd/rollback.go

```diff
@@ -54,8 +62,61 @@ func NewCmdRollback(fullName string, f *clientcmd.Factory, out io.Writer) *cobra
 		Long:    rollbackLong,
 		Example: fmt.Sprintf(rollbackExample, fullName),
 		Run: func(cmd *cobra.Command, args []string) {
-			err := RunRollback(f, out, cmd, args, rollback)
+			// Validate arguments
+			if len(args) == 0 || len(args[0]) == 0 {
+				cmdutil.CheckErr(cmdutil.UsageError(cmd, "A deployment name is required."))
+			}
+
+			// Extract arguments
+			format := cmdutil.GetFlagString(cmd, "output")
+			template := cmdutil.GetFlagString(cmd, "template")
+			dryRun := cmdutil.GetFlagBool(cmd, "dry-run")
+
+			// Get globally provided stuff
+			namespace, err := f.DefaultNamespace()
+			cmdutil.CheckErr(err)
+			oClient, kClient, err := f.Clients()
+			cmdutil.CheckErr(err)
+
+			// Set up the rollback config
+			rollback.Spec.From.Name = args[0]
+
+			// Make a helper and generate a rolled back config
+			helper := newHelper(oClient, kClient)
+			config, err := helper.Generate(namespace, rollback)
+			cmdutil.CheckErr(err)
+
+			// If this is a dry run, print and exit
+			if dryRun {
+				err := helper.Describe(config, out)
+				cmdutil.CheckErr(err)
+				return
+			}
+
+			// If an output format is specified, print and exit
+			if len(format) > 0 {
+				err := helper.Print(config, format, template, out)
+				cmdutil.CheckErr(err)
+				return
+			}
+
+			// Perform the rollback
+			rolledback, err := helper.Update(config)
 			cmdutil.CheckErr(err)
+
+			// Notify the user of any disabled image triggers
+			fmt.Fprintf(out, "#%d rolled back to %s\n", rolledback.LatestVersion, rollback.Spec.From.Name)
+			for _, trigger := range rolledback.Triggers {
+				disabled := []string{}
+				if trigger.Type == deployapi.DeploymentTriggerOnImageChange && !trigger.ImageChangeParams.Automatic {
+					disabled = append(disabled, trigger.ImageChangeParams.From.Name)
+				}
+				if len(disabled) > 0 {
+					reenable := fmt.Sprintf("%s deploy %s --enable-triggers", fullName, rolledback.Name)
+					fmt.Fprintf(out, "Warning: the following images triggers were disabled: %s\n", strings.Join(disabled, ","))
```

Comment:
This is what should be Stderr.  Also do it as one fprintf with newline and indent the second line by 2 spaces